sdk/service/service.go

```diff
@@ -67,9 +66,9 @@ func (s *Service) Create(srv *service.Service) (*service.Service, error) {
 	}
 	srv.Hash = hash.Hash(h)
 
-	// check if service already exists.
+	// check if service already exists and retrun it.
 	if _, err := s.serviceDB.Get(srv.Hash); err == nil {
-		return nil, &AlreadyExistsError{Hash: srv.Hash}
+		return srv, nil
```

Comment:
i think an error should be returned anyway. the modification you want to do is changing the behavior of this function.
You should do something like instance, with instance.Exist and returns an error:
https://github.com/mesg-foundation/engine/pull/1254/files#diff-8e12c6dba187b74b4392823599528018R90

The same can be applied to workflow,